### PR TITLE
Binance Private End Point

### PR DESCRIPTION
**Pull request by leofisG**
Added:
- balance & position channel
- order update channel

### CODE REVIEW

File: binancefutures-api/src/main/java/io/contek/invoker/binancefutures/api/websocket/UserWebSocketChannel.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  protected SubscriptionState subscribe(WebSocketSession session) {
+    return SubscriptionState.SUBSCRIBED;
+  }
+
+  // We do no action in unsubscription phase since all the user-related channel shared the same
+  // underlying web socket connection, and there is no way to unsubscribe to a given topic. Either
+  // all user related events will be pushed to us, or the connection is closed altogether.
+  @Override
+  protected SubscriptionState unsubscribe(WebSocketSession session) {
+    return SubscriptionState.UNSUBSCRIBING;
```
Comment (beijunyi) [inline binancefutures-api/src/main/java/io/contek/invoker/binancefutures/api/websocket/UserWebSocketChannel.java:25]: Maybe this shall be UNSUBSCRIBED?

File: binancefutures-api/src/main/java/io/contek/invoker/binancefutures/api/websocket/WebSocketMessageParser.java [hunk 2]
```diff
@@ -35,6 +40,9 @@ public AnyWebSocketMessage parse(String text) {
     if (obj.has("stream")) {
       return toStreamData(obj);
     }
+    if (obj.has("e")) {
```
Comment (beijunyi) [inline binancefutures-api/src/main/java/io/contek/invoker/binancefutures/api/websocket/WebSocketMessageParser.java:43]: Given that binance websocket for market data and websocket for user data have absolutely no overlap. Maybe we shall have 2 parsers: UserWebSocketMessageParser, MarketWebSocketMessageParser?

**Review by pimpcapital — APPROVED** [review]
Nicely done. Please address the comments after merge.

File: binancefutures-api/src/main/java/io/contek/invoker/binancefutures/api/websocket/UserWebSocketChannel.java (hunk 1 above)
Comment (524119574) [reply binancefutures-api/src/main/java/io/contek/invoker/binancefutures/api/websocket/UserWebSocketChannel.java]: Changed.

File: binancefutures-api/src/main/java/io/contek/invoker/binancefutures/api/websocket/WebSocketMessageParser.java (hunk 2 above)
Comment (524119574) [reply binancefutures-api/src/main/java/io/contek/invoker/binancefutures/api/websocket/WebSocketMessageParser.java]: This is done.

**Pull request merged**